src/pages/library/DataBrowserToggler.js

```diff
@@ -97,30 +89,23 @@ export const DataBrowserPreviewToggler = ({ checked }) => {
         style: { fontWeight: 'bold', display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: 6 }
       }, [
         h(Switch, {
-          checked,
+          checked: catalogShowing,
           onLabel: '', offLabel: '',
           width: 55, height: 25,
-          onChange: () => {
-            catalogPreviewStore.set({ [id]: !checked })
-            if (checked) {
-              Nav.goToPath('library-datasets')
-            } else {
-              Nav.goToPath('library-browser')
-            }
-          }
+          onChange
         }),
-        div({ style: { marginLeft: 10 } }, [`BETA Data Catalog ${checked ? 'ON' : 'OFF'}`])
+        div({ style: { marginLeft: 10 } }, [`New Catalog ${catalogShowing ? 'ON' : 'OFF'}`])
       ])
     ]),
-    checked && div({ style: { display: 'flex' } }, [
+    catalogShowing && div({ style: { display: 'flex' } }, [
       icon('talk-bubble', { size: 45, style: { marginLeft: '1.5rem', margin: '0 0.5rem' } }),
       div({ style: { display: 'flex', flexDirection: 'column' } }, [
-        strong('Provide feedback'),
+        'Provide feedback or looking for a dataset not listed?',
         h(Link, {
           onClick: () => setFeedbackShowing(true),
           style: { display: 'block', marginTop: 10 }
         },
-        ['What do you think about the new Data Catalog?'])
+        ['Let us know!'])
```

Comment:
It would be nice to add an `aria-label` to this link since "Let us know!" isn't descriptive without the context of "Provide feedback or looking for a dataset not listed?".